test/source-sink-errors/mzcompose.py

```diff
@@ -264,12 +287,14 @@ def assert_recovery(self, c: Composition) -> None:
         breakage=lambda c, _: c.pause("redpanda"),
         expected_error="OperationTimedOut|BrokerTransportFailure|transaction",
         fixage=lambda c, _: c.unpause("redpanda"),
+        expected_hint="single",
```

Comment:
@bkirwi Does this make sense? Since I added the hint for `OperationTimedOut`, it does show up for this particular disruption.

Reply:
Please note that messing with the Redpanda can cause any of the `expected_error` parts above to happen. So the `hint` must be valid for all. You may wish to run that test in a while loop for a while to confirm that it continues to pass reliably:

```
while bash -c -e "./mzcompose down -v ; ./mzcompose run default " ;  do : ; done
```

Reply:
Oh I see. Yeah, let me do that. This particular hint only maps to `OperationTimedOut`.

Reply:
I ran the while loop, but could not get a failure. I think it would be much safer to pair the error and the corresponding hint string. Let me see if I can refactor the code to do that instead

Reply:
Since I added a separate disruption case, removed this change of passing the expected_hint from here cc @guswynn @bkirwi @philip-stoev

Reply:
Since I added the separate disruption, removed the previous Redpanda specific changes. cc @bkirwi @guswynn @philip-stoev